src/entities/schedule/hooks/use-detailed-schedule.ts

```diff
@@ -0,0 +1,13 @@
+import { useSuspenseQuery } from '@tanstack/react-query'
+
+import { getDetailedSchedule } from '../models/detailed-schedule.API'
+import { scheduleQueryKeys } from '../models/schedule.query'
```

Comment:
지금 pr에서 하라는건 아니고용 나중에 배럴 패턴 적용해서 impor문 한줄로 줄여주세요!
```suggestion
import { getDetailedSchedule, scheduleQueryKeys } from '../models'
```

Reply:
넵!